src/usuarios/emails.js

```diff
@@ -1,17 +1,41 @@
 const nodemailer = require('nodemailer');
 const jwt = require('jsonwebtoken');
 
-class Email {
-  async enviaEmail() {
-    const contaTeste = await nodemailer.createTestAccount();
+class ConfiguracaoEmail {
+  constructor() {
+    this.host = process.env.EMAIL_HOST;
+    this.auth = {
+      user: process.env.EMAIL_USUARIO,
+      pass: process.env.EMAIL_SENHA
+    };
+  }
+}
 
-    const transportador = nodemailer.createTransport({
-      host: 'smtp.ethereal.email',
-      auth: contaTeste
-    });
+class ConfiguracaoEmailTeste {
+  constructor(contaTeste) {
+    this.host = 'smtp.ethereal.email';
+    this.auth = contaTeste;
+  }
+}
 
+async function criaConfiguracaoEmail() {
+  if (process.env.NODE_ENV === 'production') {
+    return new ConfiguracaoEmail();
+  } else {
+    const contaTeste = await nodemailer.createTestAccount();
+    return new ConfiguracaoEmailTeste(contaTeste);
+  }
+}
+
+class Email {
+  async enviaEmail() {
+    const configuracaoEmail = await criaConfiguracaoEmail();
+    const transportador = nodemailer.createTransport(configuracaoEmail);
     const info = await transportador.sendMail(this);
-    console.log('URL:' + nodemailer.getTestMessageUrl(info));
+    // Tem como mandar imprimir isso apenas em desenvolvimento sem usar if?
```

Comment:
No geral em produção temos uma regra de linter que proíbe console.log. Provavelmente isso cairia em um teste de integração e se tivesse errado o teste avisaria